packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -25,7 +26,9 @@ const getFilteredItems = (val, list, should_filter_by_char) => {
     }
 
     return list.filter(item =>
-        is_string_array ? item.toLowerCase().includes(val) : item.text.toLowerCase().includes(val)
+        is_string_array
+            ? item.toLowerCase().includes(val)
```

Comment:
should'nt we add this string normalization here when is_string_array is true?

Reply:
Incorporated this